Source/WebKit/GPUProcess/media/RemoteSourceBufferProxy.messages.in

```diff
@@ -61,4 +61,6 @@ messages -> RemoteSourceBufferProxy NotRefCounted {
     MinimumUpcomingPresentationTimeForTrackID(WebCore::TrackID trackID) -> (MediaTime time) Synchronous
 }
 
+using WebCore::SourceBufferPrivate::SamplesPromise::Result = Expected<Vector<String>, int>
```

Comment:
What is this for?

Reply:
BufferedSamplesForTrackId and EnqueuedSamplesForTrackID message responses

Reply:
Which generated file will have this "using" statement? I tried making the change and building locally, and don't see it in RemoteSourceBufferProxyMessages.h/cpp (and messages.py does not seem to add using statement).

Reply:
SerializedTypeInfo.mm

Reply:
Oh, you're right.  This is in a bad place.  I'm moving it to WebCoreArgumentCoders.serialization.in